### PR TITLE
fselect: init at 0.6.6

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/73044)

### CODE REVIEW

File: pkgs/tools/misc/fselect/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  src = fetchFromGitHub {
+    owner = "jhspetersson";
+    repo = "fselect";
+    rev = version;
+    sha256 = "0lwh0qcyy9hmkxijlhp0inznrb0ibw1wcnnlqfxc71y4ar42f165";
+  };
+
+  cargoSha256 = "0s7yld3cwmyfdsqwcipc2l07xx5qn3knjmh4bbnjmnxiygaiszx9";
+
+  cargoPatches = [ ./cargo-lock.patch ];
```
Comment (zimbatm) [inline pkgs/tools/misc/fselect/default.nix]: is it possible to ask upstream to make a new release? the patch file is rather large and probably the result of upstream not checking in the latest version of it.

File: pkgs/tools/misc/fselect/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  meta = with stdenv.lib; {
+    description = "Find files with SQL-like queries";
+    homepage = "https://github.com/jhspetersson/fselect";
+    license = with licenses; [ asl20 /* or */ mit ];
+    maintainers = [ ];
```
Comment (zimbatm) [inline pkgs/tools/misc/fselect/default.nix]: can you add yourself as the maintainer?

**Comment by filalex77** [comment]
Would be great, but I already have a PR adding me to the maintainer list (for `ruplacer`). Is it okay for me to duplicate these changes here?

File: pkgs/tools/misc/fselect/default.nix (hunk 2 above)
Comment (filalex77) [reply pkgs/tools/misc/fselect/default.nix]: Would be great, but I already have a PR adding me to the maintainer list (for `ruplacer`). Is it okay for me to duplicate these changes here?

**Comment by filalex77** [comment]
Closed in favor of #73206.

Changes:
- added myself as a maintainer
- updated to 0.6.7
- removed the patch (`Cargo.lock` added upstream in jhspetersson/fselect#61)

**Pull request closed without merging**